Formula/lib/libaec.rb

```diff
@@ -27,12 +27,10 @@ class Libaec < Formula
   link_overwrite "lib/libsz.so.2"
 
   def install
-    mkdir "build" do
-      # We run `make test` for libraries
-      system "cmake", "..", *std_cmake_args, "-DBUILD_TESTING=ON"
-      system "make", "install"
-      system "make", "test"
-    end
+    system "cmake", "-S", ".", "-B", "build", "-DBUILD_TESTING=ON", *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DBUILD_TESTING=ON"
```
`std_cmake_args` includes `-DBUILD_TESTING=OFF`.